admin_tests/metrics/test_utils.py

```diff
@@ -181,3 +183,51 @@ def test_get_all_user_count(self):
     def test_get_unregistered_users(self):
         count = get_unregistered_users()
         nt.assert_equal(count, 1)
+
+
+class TestRenderToCSVResponse(AdminTestCase):
+
+    def setUp(self):
+        super(TestRenderToCSVResponse, self).setUp()
+        self.app = TestApp(test_app)
+        Node.remove()
+        time_now = get_previous_midnight()
+        NodeFactory(category='project', date_created=time_now)
+        NodeFactory(category='project',
+                    date_created=time_now - timedelta(days=1))
+        last_time = time_now - timedelta(days=2)
+        NodeFactory(category='project', date_created=last_time)
+        NodeFactory(category='project', date_created=last_time)
+        initial_time = last_time + timedelta(seconds=1)
+        get_days_statistics(initial_time)
+        midtime = last_time + timedelta(days=1, seconds=1)
+        self.time = time_now + timedelta(seconds=1)
+
+        self.initial_static = [
+            'id,users,delta_users,unregistered_users,projects,delta_projects,public_projects,'
+            'delta_public_projects,registered_projects,delta_registered_projects,date\r',
+            '1,0,0,0,0,0,0,0,0,0,' + initial_time.strftime('%Y-%m-%d %H:%M:%S.%f') + '\r', '']
+        self.latest_static = [
+            'id,users,delta_users,unregistered_users,projects,delta_projects,public_projects,'
+            'delta_public_projects,registered_projects,delta_registered_projects,date\r',
+            '3,0,0,0,0,0,0,0,0,0,' + self.time.strftime('%Y-%m-%d %H:%M:%S.%f') + '\r',
```

Comment:
You also didn't respond to the part about having values for things like `public_projects` in https://github.com/CenterForOpenScience/osf.io/pull/6288#discussion_r80723828.